src/SprykerEco/Zed/PunchoutCatalogs/Presentation/EntryPoints/view.twig

```diff
@@ -0,0 +1,35 @@
+{% extends '@Gui/Layout/layout.twig' %}
+
+{% set widget_title = 'Entry Points for Connection:' | trans ~ ' ' ~ punchoutCatalogConnection.name %}
+
+{% block head_title widget_title | trans %}
+{% block section_title widget_title | trans %}
+
+{% block action %}
+    {{ backActionButton('/punchout-catalogs', 'List of PunchOut connections' | trans) }}
+{% endblock %}
+
+{% block content %}
+    {% embed '@Gui/Partials/widget.twig' with { widget_title: 'Entry Points' } %}
+        {% block widget_content %}
+            <table class="table table-bordered">
+                <thead>
+                    <tr>
+                        <th>{{ 'Store Name' | trans }}</th>
+                        <th>{{ 'URL Entry Point' | trans }}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for punchoutCatalogEntryPoint in punchoutCatalogEntryPoints %}
```

Comment:
Please add if condition before loop

Reply:
Discussed with @gerner-spryker, we always have at least 1 entry point

![Screen Shot 2019-06-26 at 6 30 59 PM](https://user-images.githubusercontent.com/15832795/60193814-f824c900-9840-11e9-8be4-63f1a234e193.png)